todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/domain/effect.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.android.architecture.blueprints.todoapp.taskdetail.domain
+
+import com.example.android.architecture.blueprints.todoapp.data.Task
+
+sealed class TaskDetailEffect {
```

Comment:
I'm guessing the cases are defined as inner classes to not break Java imports? I'd rather we don't define them as inner classes as such and follow the same patter from the Add/Edit Task feature